frontend/apps/app/components/ProjectSessionsPage/ProjectSessionsPage.tsx

```diff
@@ -19,39 +19,41 @@ export const ProjectSessionsPage: FC<Props> = async ({ projectId }) => {
 
   return (
     <div className={styles.container}>
-      <div className={styles.header}>
-        <div className={styles.titleSection}>
-          <h2 className={styles.title}>Sessions</h2>
-          <p className={styles.description}>Design sessions for this project</p>
-        </div>
-      </div>
-
-      <div className={styles.formSection}>
-        <h3 className={styles.formTitle}>Create New Session</h3>
+      <div className={styles.wrapper}>
+        <h1 className={styles.title}>
+          Create a new session for
+          <br />
+          this project
```

Comment:
I was concerned about the line break position. Personally, I think the following is more natural.🤔💭

```suggestion
          Create a new session
          <br />
          for this project
```